src/main/java/domain/Mark.java

```diff
@@ -0,0 +1,26 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum Mark {
+    HEART("하트"),
+    DIAMOND("다이아몬드"),
+    SPADE("스페이드"),
+    CLOVER("클로버");
```

Comment:
도메인 용어를 잘 지키는 것도 중요합니다.
저도 미션을 진행하면서 알았는데 `CLUBS` 를 사용하더라구요.
추가로 복수로 부르는 것이 일반적이라고 합니다.

```suggestion
public enum Mark {
    HEARTS("하트"),
    DIAMONDS("다이아몬드"),
    SPADES("스페이드"),
    CLUBS("클로버");
```